cluster/defaults.go

```diff
@@ -220,13 +221,15 @@ func (c *Cluster) setClusterDefaults(ctx context.Context, flags ExternalFlags) e
 
 func (c *Cluster) setNodeUpgradeStrategy() {
 	if c.UpgradeStrategy == nil {
-		logrus.Infof("No input provided for maxUnavailable, setting it to default value of %v", DefaultMaxUnavailable)
+		logrus.Debugf("No input provided for maxUnavailable, setting it to default values of %v", DefaultMaxUnavailableWorker, DefaultMaxUnavailableControlplane)
```

Comment:
Please describe this as what value for controlplane and what value for workers, there is also only one value to be replaced in the string while 2 values are passed